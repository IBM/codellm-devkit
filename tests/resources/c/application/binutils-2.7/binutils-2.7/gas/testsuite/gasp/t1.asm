
	test for eof in middle of line
	.END
