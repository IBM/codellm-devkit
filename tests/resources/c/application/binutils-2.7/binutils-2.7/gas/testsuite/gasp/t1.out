!

!	test for eof in middle of line
	test for eof in middle of line
!	.END
